lib/facade/FileClient.ts

```diff
@@ -36,9 +36,7 @@ export class FileClient {
     source: ReadableStream | Buffer | string | ArrayBuffer,
     params: EncryptParamsBuilder | DecryptParamsBuilder
   ) {
-    // @ts-ignore
-    if (source.pipe !== undefined || source instanceof ReadableStream) {
```

Comment:
source?.pipe   ?

Reply:
Actual problem is that TS doesn't understand what type is it.  So with optional operator also will get error warn